type-message.go

```diff
@@ -42,13 +44,25 @@ func messageFromFile(context *context) (*message, error) {
 	var bodyLine *line
 	var bodyLines []*line
 
+	duration := time.Duration(0)
+
 	for {
 		bodyLine, err = newLineFromFile(context)
 
 		if err == io.EOF || bodyLine.isBlank() {
 			break
 		}
 
+		if bodyLine.isSleep() {
+			duration, err = time.ParseDuration(bodyLine.Text)
+
+			if err != nil {
+				return nil, err
+			}
+
+			break
+		}
+
 		if err != nil {
 			return nil, err
 		}
```

Comment:
isn't this err check for line 50? if so then it should be above the isSleep check.